src/commands/buildModule.ts

```diff
@@ -34,6 +34,14 @@ export async function buildModule (node?: DeviceNode | OSVerNode | PlatformNode
 			scope: vscode.TaskScope.Workspace
 		};
 
+		if (node?.targetId) {
+			taskDefinition.definition.titaniumBuild.target = node.targetId as 'device' | 'emulator' | 'simulator';
+		}
+
```

Comment:
I think also add the below here so that if an iOS version node is chosen in the explorer then it will use that info too 

```js
if (node?.version) {
	taskDefinition.definition.titaniumBuild.ios = { simulatorVersion: node.version };
}
```